### PR TITLE
add configuration option for flags

### PR DESCRIPTION
**Pull request by Kasi07**
As I have two accounts in my 1Password vault, I couldn't receive any secrets.
So I added a “flags” config to configure [1Password global flags](https://developer.1password.com/docs/cli/reference/#global-flags).

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import { validateCli, read } from '@1password/op-js';
+import { validateCli, read, setGlobalFlags } from '@1password/op-js';
 import * as cache from './cache';
 import path from 'path';
 import fs from 'fs';
 
 type PluginConfig = {
   cliPath?: string;
+  flags?: {};
```
Comment (benvp) [inline src/app.ts]: Can you change that type to e.g. `Record<string, any>`? Using `{}` as a type means that this is an object without any keys assigned to it.

**Review by benvp — CHANGES_REQUESTED** [review]
Tahnks! Just a minor thing.

In addition, we should look into how this is possible when using https://github.com/benvp/insomnia-plugin-op/issues/2 the one password sdk

**Comment by shadi-sharaf-nuk** [comment]
Thanks [Kasi07](https://github.com/Kasi07)! I was just going to PR the change myself and I'm happy to have checked the pull-requests first :D 

Thanks @benvp for the quick review, would really love to have this change released!

**Review by benvp — APPROVED** [review]

**Comment by benvp** [comment]
Sorry for the delay. Didn't have any time yet. Just released 0.2.0. And thanks for your contribs!

**Pull request merged**